web/src/routes/TagDetails/TagDetailsTabs.tsx

```diff
@@ -59,6 +61,12 @@ export default function TagTabs(props: TagTabsProps) {
       >
         <Packages />
       </Tab>
+      <Tab
+        eventKey={TabIndex.ModelCard}
+        title={<TabTitleText>ModelCard</TabTitleText>}
```

Comment:
Can we also feature flag this? This will require additional work to get running on quay.io